id_minter/src/main/scala/uk/ac/wellcome/platform/idminter/steps/IdentifierGenerator.scala

```diff
@@ -32,8 +35,9 @@ class IdentifierGenerator @Inject()(identifiersDao: IdentifiersDao,
   }
 
   private def retrieveOrGenerateCanonicalId(
-    identifier: SourceIdentifier): Future[String] =
-    identifiersDao.lookupMiroID(identifier.value).flatMap {
+    identifier: SourceIdentifier,
+    ontologyType: String): Future[String] = {
+    identifiersDao.lookupID(List(identifier), ontologyType).flatMap {
```

Comment:
This function really needs another level of refactoring, so it takes the list of source identifiers rather than a single source identifier – but this patch is just about ripping out `lookupMiroID`.